dependency.lic

```diff
@@ -196,7 +196,7 @@ class ScriptManager
     unless LICH_VERSION.match?(/^5/)
       _respond("<pushBold/>*****************************************************************************<popBold/>")
       _respond("<pushBold/>* Unsupported Lich versions detected.                                       *<popBold/>")
-      _respond("<pushBold/>* Please see https://github.com/rpherbig/dr-scripts/wiki/First-Time-Setup   *<popBold/>")
+      _respond("<pushBold/>* See https://github.com/elanthia-online/dr-scripts/wiki/First-Time-Setup   *<popBold/>")
```

Comment:
There are also a number of URLs which could be updated in other files: https://github.com/search?q=repo%3Aelanthia-online%2Fdr-scripts%20rpherbig&type=code

Reply:
Good reminder. Thanks!

https://github.com/elanthia-online/dr-scripts/pull/7001 addresses all of these.